arborist/server_test.go

```diff
@@ -1105,6 +1105,35 @@ func TestServer(t *testing.T) {
 			getResourceWithPath(t, "/Godel,/completeness_theorem")
 		})
 
+		t.Run("Merge", func(t *testing.T) {
```

Comment:
we could maybe add one more test or command in here that updates an existing subresource instead of top level. So like update the `dog` resource, add `goldenretriever` or something. What's the behavior if you do that but leave out `corgi`? I assume it also stays around?

Reply:
Yeah the expected behavior would be that `corgi` stays around.